### DISCUSSION TITLE
Using local font in Nodejs

### DISCUSSION THREAD

**Discussion by kaisiant** [diegomura/react-pdf#2139] [Q&A] [answered]
Currently Im using the cdn source but I want to change it to using local source. How can I do it? As I facing the error. "`wOFFOTTO SyntaxError: Invalid or unexpected token`"

<img width="766" alt="image" src="https://user-images.githubusercontent.com/26655111/211716980-aefad01d-3536-4fb5-bffa-f67c3f294a11.png">
<img width="734" alt="image" src="https://user-images.githubusercontent.com/26655111/211717074-29a8388c-a448-45b0-9340-ea7caa9da483.png">

**Comment by kaisiant** [comment, marked as answer]
Just need to remove the `require` :)

**Comment by TheBlindHawk** [comment]
Was it different in previous versions?
All comments I found in the Issues I read seemed to point at the fact that you need to use ```import``` or ```require```.

Also here is an example if you are using ```path``` (in case you get ```no such file or directory```)
```typescript
  import path from 'path'

  Font.register({
    family: "NotoSansJP-Regular",
    format: "truetype",
    src: path.join(__dirname, './NotoSansJP-Regular.ttf'),
  });
```
